Changed footer logos and home page text

momo3404 opened the pull request:
Fixes #1057 .

Changes proposed in this PR:
- Adjusted the logo ordering and added the UVic and Arbutus logos. The images had to be resized to fit properly on the footer.
- Added the appropriate text to the home page.

aaronskiba committed:
Delete `tinymce/skins` & add to .gitignore

`tinymce/skins` and its contents are auto-generated (see `config/initializers/assets.rb`).

aaronskiba committed:
Add logic allowing superusers to confirm emails

This change addresses an issue where some users reported not receiving the confirmation instructions email. Rather than having to manually update the users email confirmation status within the Rails console, this change allows users to confirm/unconfirm user emails within the app.

aaronskiba committed:
Add users_controller_spec + confirmation tests

aaronskiba committed:
Fix email confirmation checkbox's alignment

aaronskiba committed:
Prevent unconfirming of super admin emails

aaronskiba committed:
Prevent superadmin un-confirmation if UI bypassed

`spec/controllers/super_admin/users_controller_spec.rb`:
- Added test to prevent un-confirming of any super admin email within app
`app/controllers/super_admin/users_controller.rb`:
- Added !@user.can_super_admin? check to enable the aforementioned test to pass
- Also, updated a `patch` statement to `put`

aaronskiba committed:
Update CHANGELOG.md

aaronskiba committed:
Update handling of default template in dropdown

This change resolves https://github.com/portagenetwork/roadmap/issues/1013
Before this change, `!customization.upgrade_customization?` was applied to all templates populating the dropdown, except the default template. As a result, if an organization had customized and published an older version of the default template, but not the newest version, that newest version would be blocked from being populated in the dropdown.

aaronskiba committed:
Update CHANGELOG.md

dependabot[bot] committed:
Bump nokogiri from 1.18.3 to 1.18.8

Bumps [nokogiri](https://github.com/sparklemotion/nokogiri) from 1.18.3 to 1.18.8.
- [Release notes](https://github.com/sparklemotion/nokogiri/releases)
- [Changelog](https://github.com/sparklemotion/nokogiri/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sparklemotion/nokogiri/compare/v1.18.3...v1.18.8)

---
updated-dependencies:
- dependency-name: nokogiri
  dependency-version: 1.18.8
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

aaronskiba committed:
`bundle lock --add-platform x86_64-linux-musl`

- Commit 33067a1c6b81fd3a166696ae47cbeffa519ade38 removes the `x86_64-linux-musl` platform. However, it is required for the Alpine Linux Docker build.
- This appears to be a reoccurring issue with Dependabot PRs that should be addressed.

aaronskiba committed:
Update CHANGELOG.md

dependabot[bot] committed:
Bump @babel/runtime from 7.21.0 to 7.27.1

Bumps [@babel/runtime](https://github.com/babel/babel/tree/HEAD/packages/babel-runtime) from 7.21.0 to 7.27.1.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.27.1/packages/babel-runtime)

---
updated-dependencies:
- dependency-name: "@babel/runtime"
  dependency-version: 7.27.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

aaronskiba committed:
Update CHANGELOG.md

dependabot[bot] committed:
Bump @babel/helpers from 7.21.0 to 7.27.1

Bumps [@babel/helpers](https://github.com/babel/babel/tree/HEAD/packages/babel-helpers) from 7.21.0 to 7.27.1.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.27.1/packages/babel-helpers)

---
updated-dependencies:
- dependency-name: "@babel/helpers"
  dependency-version: 7.27.1
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

aaronskiba committed:
Update CHANGELOG.md

dependabot[bot] committed:
Bump rack from 2.2.13 to 2.2.14

Bumps [rack](https://github.com/rack/rack) from 2.2.13 to 2.2.14.
- [Release notes](https://github.com/rack/rack/releases)
- [Changelog](https://github.com/rack/rack/blob/main/CHANGELOG.md)
- [Commits](https://github.com/rack/rack/compare/v2.2.13...v2.2.14)

---
updated-dependencies:
- dependency-name: rack
  dependency-version: 2.2.14
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

aaronskiba committed:
Update CHANGELOG.md

dependabot[bot] committed:
Bump net-imap from 0.4.19 to 0.4.20

Bumps [net-imap](https://github.com/ruby/net-imap) from 0.4.19 to 0.4.20.
- [Release notes](https://github.com/ruby/net-imap/releases)
- [Commits](https://github.com/ruby/net-imap/compare/v0.4.19...v0.4.20)

---
updated-dependencies:
- dependency-name: net-imap
  dependency-version: 0.4.20
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

aaronskiba committed:
Update CHANGELOG.md

aaronskiba committed:
Match staging image tags to new release candidate

sbathgate committed:
feat(secrets): add sealed secrets for cephfs locales and uploads

sbathgate committed:
fix(config): update Rollbar environment for production

sbathgate committed:
fix ceph secrets

sbathgate committed:
fix(secrets): update encrypted keys for cephfs locales and uploads

sbathgate committed:
fix(secrets): update cephfs mount user for locales and uploads

sbathgate committed:
fix(storage): update paths for locales and uploads in cephfs configuration

aaronskiba committed:
Add dev `integrations` and `devise` sealed secrets

- `deployments/dev/sealed-secrets/db-sealed-secret.yaml` still needs to be added

aaronskiba committed:
Fix naming `-secret-sealed` -> `-sealed-secret`

aaronskiba committed:
Create db-sealed-secret.yaml

sbathgate committed:
fix(secrets): update cephfs user naming convention with domain prefix

sbathgate committed:
fix(deployment): change issuer to letsencrypt-dev-cloudflare

sbathgate committed:
chore(storage): update cephfs user definition to handle missing domain prefix

momo3404 committed:
Changed footer logos and home page text

momo3404 commented:
This is what the footer looks like now:
![image](https://github.com/user-attachments/assets/96075c70-c235-4c31-9b1f-198040bae138)

This is what the new home page text looks like:
![image](https://github.com/user-attachments/assets/b996540d-ab34-4fa6-9d39-9e5827108f97)

momo3404 committed:
edited changelog.md

momo3404 committed:
edited changelog.md

aaronskiba commented on app/views/layouts/_footer.html.erb:
```diff
@@ -20,44 +20,47 @@
       <li>
-        <%= link_to("https://www.library.ualberta.ca/", title: "University of Alberta Library", target: '_blank', class: 'has-new-window-popup-info')  do %>
-          <%= image_tag("UA-LIBR-1C-SOLID-REVERSE.png",
+        <%= link_to("https://www.uvic.ca/", title: _("University of Victoria"), target: '_blank', class: 'has-new-window-popup-info')  do %>
```
I like that you're using `_()` for the various `title:` values (e.g. `title: _("University of Victoria")`.
It looks like `_()` was missing in some places prior to your changes (e.g. `title: "DMP Online"`). We should probably address those missing places as part of this PR (e.g. `title: "University of Alberta Library"` -> `title: _("University of Alberta Library")`, so that any possible translations are enabled.

momo3404 replied:
Sounds good, I'll get on that

aaronskiba committed:
Add comment to `def handle_confirmed_at_param`

aaronskiba committed:
Clarify email confirmation checkbox instructions

Improves UX and reduces confusion when updating a user's
email confirmation status.

- Moved confirmation status inline with the "Email status:" heading
- Updated checkbox helper text to reflect current @user.confirmed? state:
  - Shows "(Check to confirm email)" if unconfirmed
  - Shows "(Uncheck to unconfirm email)" if confirmed

aaronskiba committed:
Capture diff when `public/tinymce/skins` is deleted and auto-generated

This commit deletes `public/tinymce/skins` and regenerates it to capture the diff.

It shows which files are no longer auto-generated (e.g., mobile skin files), and can be used to help verify the impact of deleting the entire `public/tinymce/skins` directory.

aaronskiba committed:
Update CHANGELOG.md

- Add entry for this PR (#1060) and fix placement of PR #1009

aaronskiba committed:
Comment dropdown handling of default template

momo3404 committed:
enabled all translations

momo3404 committed:
Edited sample message to match feedback message in text box

momo3404 committed:
Undo accidental last commit
